java-chassis-dependencies/pom.xml

```diff
@@ -1003,6 +1009,16 @@
 			</dependency>
 			<dependency>
 				<groupId>io.servicecomb</groupId>
+				<artifactId>foundation-crypt</artifactId>
+				<version>0.1.0-m2-SNAPSHOT</version>
```

Comment:
the conflict resolving with master is not wring. these dependencies were missing and removed on master.